src/CanvasClass/CanClass.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+class CanClass extends React.Component {
+    componentDidMount() {
+      const canvas = this.refs.canvas
+      const ctx = canvas.getContext("2d")
+      const img = this.refs.image
+      ctx.fillStyle = "#3ab2d6";
+      ctx.fillRect(0,0,200,90)
+      ctx.fillStyle = "#c4d7dd";
```

Comment:
You set the fillStyle twice here. Once as `#3ab2d6` at line 8. Then here as `#c4d7dd`. React will use the one you set last. Which means line 8 is never used, you can remove it.

Reply:
No, I don't really think so. I'm kind of drawing two different things on top of each other, so the first color, it's for drawing the rectangle onto the canvas in color #3ab2d6. The second one is to "draw" the text on top of the rectangle which is the color #c4d7dd. If you would just try to delete the line 8 you'd get a black rectangel which is the default drawing color, and if you'd try to remove the line 10 you won't be able to even see the text because it'll be drawned in the same color of the rectangle.
Sorry for the late reply, but i was really sick.

Reply:
You're right, my bad. I don't have much experience with canvas, didn't know you could have two fillstyles.